OpenRA.Game/Support/PerfTimer.cs

```diff
@@ -24,8 +24,9 @@ public sealed class PerfTimer : IDisposable
 		const string IndentationString = "|   ";
 		const string FormatSeperation = " ms ";
 		static readonly string FormatString = "{0," + Digits + ":0}" + FormatSeperation + "{1}";
+		static readonly string FormatStringLongTick = "{0," + Digits + ":0}" + FormatSeperation + "[{1}] {2}: {3}";
```

Comment:
These should probably get translations

Reply:
I believe it will only end up in the performance.log.

If so, let's do in in a seperate pull request.

Reply:
Yeah I agree that we don't need a translation if it just lands in logs.